sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LogicalPlan.scala

```diff
@@ -381,6 +381,15 @@ object LogicalPlanIntegrity {
     }.flatten
   }
 
+  def validateSchemaOutput(previousPlan: LogicalPlan, currentPlan: LogicalPlan): Option[String] = {
+    if (!DataTypeUtils.equalsIgnoreNullability(previousPlan.schema, currentPlan.schema)) {
+      Some(s"The plan output schema has changed from ${previousPlan.schema.sql} to " +
```

Comment:
I think we can use string block syntax instead of manually adding `\n` for new lines.

Reply:
Yes, we can do that. The current style is consistent with other validation functions so if we want to switch to string block syntax, I think we should change other validation functions, too. I don't have a strong opinion on this, please let me know if that's preferred.

Reply:
This is not a very important issue, so it's not worth making batch changes. It's just that this pr touched a related case, so I hope it can be conveniently fixed. Personally, I think the string block syntax  is relatively clearer, for example, there is no need to pay attention to the escape characters :)